umm-spec-lib/src/cmr/umm_spec/migration/version/granule.clj

```diff
@@ -96,6 +96,30 @@
              util/remove-nil-keys))
        g))
 
+(defn downgrade-format-and-mimetype-to-1-6
+  "This function takes a map and downgrades the DMRPP format and
+   mime-type to version 1.6. If the metadata contains :Files then it is a nested map and so call
+   this function again to iterate over its maps.
+   An updated map is returned."
+  [archive-and-dist-info-file]
+  (-> archive-and-dist-info-file
+      (update :Format #(if (= "DMRPP" %)
+                          "Not provided"
+                          %))
+      (update :MimeType #(if (= "application/vnd.opendap.dap4.dmrpp+xml" %)
+                           "Not provided"
+                           %))
```

Comment:
You can probably use `string/replace` here instead to simplify this a little bit

https://clojuredocs.org/clojure.string/replace

Reply:
The problem with replace is that it can't take a nil string. An if check is required to make sure that a string exists and then do the replace. Once that is done then the code isn't any simpler.  I will leave it as it is.